python/pyspark/pandas/series.py

```diff
@@ -507,7 +517,16 @@ def add(self, other: Any) -> "Series":
         series_examples=_add_example_SERIES,
     )
 
-    def radd(self, other: Any) -> "Series":
+    def radd(self, other: Any, fill_value: Union[int, str, float] = None) -> "Series":
+        if fill_value is not None:
+            if isinstance(other, (int, str, float)):
+                new_sdf = self._internal.resolved_copy.spark_frame.fillna(fill_value)
```

Comment:
Hm, can we actually use an expression instead of DataFrame API? This will require to turn on the operation between different DataFrames. e.g., `col = F.when(lag_col.isNull() | F.isnan(lag_col), fill_value).otherwise(lag_col)`

Reply:
Sounds good! Let me refine soon.

Thanks!

Reply:
Updated!